src/table.mjs

```diff
@@ -231,3 +271,91 @@ function likeOperand(operand) {
   return {...operand, value: `%${operand.value}%`};
 }
 
+// This function applies table cell operations to an in-memory table (array of objects).
+export function __table(source, operations) {
+  let {schema, columns} = source;
+  for (const {type, operands} of operations.filter) {
+    const column = operands.find(({type}) => type === "column").value;
+    const resolved = operands.filter(({type}) => type === "resolved");
```

Comment:
I’m not sure this is right. According to our schema, TableFilterOperands can have the following type:

* _primitive_
* _date_
* _cell_

I don’t see us using the type _resolved_ anywhere. I’m also not sure what the type _cell_ means… I’m guessing that was speculative and should be removed from the schema. Anyway, I think we can say this:

```suggestion
    const values = operands.slice(1).map(({value}) => value);
```

Reply:
Ah ha. The compiler replaces _primitive_, _date_, and _cell_ filter operands with _resolved_. So it was correct as-is. I’ll update my PR accordingly.